org.eclipse.lsp4j.debug/src/main/java/org/eclipse/lsp4j/debug/DebugProtocol.xtend

```diff
@@ -4227,3 +4236,38 @@ interface InvalidatedAreas {
 	 */
 	public static final String VARIABLES = "variables";
 }
+
+/**
+ * Describes how the `startDebugging` request should be handled
+ */
+enum StartDebuggingRequestType {
+	LAUNCH,
+	ATTACH
+}
+
+/**
+ * Arguments for <code>startDebugging</code> reverse-request
+ */
+@JsonRpcData
+class StartDebuggingRequestArguments {
+
+	/**
+	 * Arguments passed to the new debug session. The arguments must only contain
+	 * properties understood by the `launch` or `attach` requests of the debug
+	 * adapter and they must not contain any client-specific properties (e.g.
+	 * `type`) or client-specific features (e.g. substitutable 'variables').
+	 */
+	@NonNull
+	Map<String, ?> configuration;
+
+	/**
+	 * Indicates whether the new debug session should be started with a `launch`
+	 * or `attach` request.
+	 */
+	@NonNull
+	StartDebuggingRequestType request;
+}
+
+@JsonRpcData
+class StartDebuggingResponse {
```

Comment:
Remove this - we use `Void` type when there is just an ack response. e.g. there is no `ConfigurationDoneResponse`